src/templates/en/2019/table_of_contents.html

```diff
@@ -156,6 +156,29 @@
 <section class="main">
   <h1 class="title">Table of Contents</h1>
 
+  <section>
+    <h2>Foreword</h2>
+
+    <p>
+      The open web is an amazingly complex, evolving network of technologies. Entire industries and careers are built on the web and depend on its vibrant ecosystem to succeed. As critical as the web is, understanding how it's doing has been surprisingly elusive. Since 2010, the mission of the <a href="https://httparchive.org/">HTTP Archive</a> project has been to track how the web is built. And it's been doing an amazing job of it. But there has been one gap that has been especially challenging to close: bringing meaning to the data and enabling the web community to easily understand how it's performing. That's where the Web Almanac comes in.
+    </p>
+
+    <p>
+      The mission of the Web Almanac is to take the treasure trove of insights that would otherwise be accessible only to intrepid data miners and package it up in a way that's easy to understand. This is made possible with the help of industry experts who can make sense of the data and tell us what it means. Each of the 20 chapters in the Web Almanac focuses on a specific aspect of the web and it's authored and peer reviewed by experts in their field. The strength of the Web Almanac flows directly from the expertise of the people who write it.
+    </p>
+
+    <p>
+      Many of the findings in the Web Almanac are are worthy of celebration, but it's also an important reminder of the work still required to deliver high quality user experiences. The data-driven analyses in each chapter are a form of accountability we all share for developing a better web. It's not about shaming those that are getting it wrong, but about shining a guiding light on the path of best practices so there is a clear right way to do things. And with the continued help of the web community, we hope to make this an annual tradition, so each year we can track our progress and make course corrections as needed.
+    </p>
+
+    <p>
+      There is so much to learn in this report, so start exploring and share your takeaways with the community so we can collectively advance our understanding of the state of the web.
+    </p>
```

Comment:
```suggestion
    </p>
    <p>
      Finally, 2019 is the first year we have run the Web Almanac, but we hope to make this an annual report allowing us to track the evolving state of the web of time. If you're interested in contributing to the 2020 edition of the Web Almanac, please fill out our <a href="https://forms.gle/Qyf3q5pKgdH1cBhq5">interest form</a>. We'd love to hear your ideas for making this project even better!
    </p>
```

Reply:
The "annual report" thing is covered in paragraph 3. The form is a good idea but I'd like to surface that differently (still thinking of how to best do this).